src/centrality.rs

```diff
@@ -78,10 +56,10 @@ where
     if graph.node_count() < parallel_threshold {
         node_indices
             .iter()
-            .map(|node_s| {
+            .map(|&node_s| {
                 (
-                    shortest_path_for_centrality(&graph, node_s),
-                    graph.to_index(*node_s),
+                    shortest_path_for_centrality(graph, node_s),
+                    graph.to_index(node_s),
```

Comment:
we can probably change this to `node_s.index()` if we wanted and probably remove the visit traits. The use of `to_index()` before was mostly because of the trait implementations, we didn't know the type of the node id so we couldn't just convert to a usize, But, stablepygraph doesn't have this problem

Reply:
Yeah but we would still need `petgraph::visit::NodeIndexable` for `graph.node_bound()`